src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -739,19 +740,29 @@ private void SearchDefaultNodeAutocompleteCandidates()
             // TODO: These are basic input types in Dynamo
             // This should be only served as a temporary default case.
             var queries = new List<string>() { "String", "Number Slider", "Integer Slider", "Number", "Boolean", "Watch", "Watch 3D", "Python Script" };
+            var categories = new List<(string, SearchElementGroup)> { (".List", SearchElementGroup.Create), (".List", SearchElementGroup.Query) };
```

Comment:
What categories of nodes would this filter include - categories that are named _something_.List?